### DISCUSSION TITLE
Combobox: don't clear the input value on outside click

### DISCUSSION THREAD

**Discussion by Yankovsky** [tailwindlabs/headlessui#1953] [Help]
Hey! I need some help adjusting the Combobox component to be more like autocomplete field and doesn't clear the value and options on click outside.

What I have currently:
https://user-images.githubusercontent.com/72094/197531380-4c210612-ca57-4115-bab4-dfe10d8b0f60.mp4
What I am aiming for (google search as an example):
https://user-images.githubusercontent.com/72094/197531832-1e57f885-dbd6-4cdd-a785-658aa9daa739.mp4

Thank you!

**Comment by hmaiermack** [comment]
Also running into this issue, did you ever solve it?

**Reply by Yankovsky** [reply]
Not really
